File: authorization-admin-service-spring/src/main/java/eu/rudisch/authorization/admin/service/UserService.java
package eu.rudisch.authorization.admin.service;

import java.util.List;

import eu.rudisch.authorization.admin.web.controller.resource.UserRestRep;

public interface UserService {
	List<UserRestRep> getUserRepresentations();

	UserRestRep updateUser(int id, UserRestRep toUpdate, String issuer);
}

File: authorization-admin-service-spring-kotlin/src/main/kotlin/eu/rudisch/authorization/admin/service/UserService.kt
package eu.rudisch.authorization.admin.service

import eu.rudisch.authorization.admin.web.controller.resource.UserRestRep

interface UserService {
	fun getUserRepresentations(): List<UserRestRep>
	fun updateUser(id: Int, toUpdate: UserRestRep, issuer: String): UserRestRep
}